aziot/test-files/init/dps-symmetric-key/identityd.toml

```diff
@@ -2,7 +2,7 @@ hostname = "my-device"
 homedir = "/var/lib/aziot/identityd"
 
 [provisioning]
-dynamic_reprovisioning = true
+always_reprovisioning_on_startup = true
```

Comment:
`always_reprovision_on_startup`, not `always_reprovisioning_on_startup`

Reply:
Fixed